cmd/root.go

```diff
@@ -196,7 +196,7 @@ func init() {
 func beginDetectNewVersion() chan struct{} {
 	completionChannel := make(chan struct{})
 	go func() {
-		const versionMetadataUrl = "https://aka.ms/azcopyv10-version-metadata"
+		const versionMetadataUrl = "https://azcopyvnextrelease.blob.core.windows.net/releasemetadata/latest_version.txt"
```

Comment:
@zezha-msft Perhaps we should figure out what went wrong with the aka.ms shortener instead of referencing the storage account directly.

Reply:
IIRC it had to do with the client request ID validation. The aka.ms link was sending back the cached request ID rather than the real request ID. 

@siminsavani-msft please update the PR description to be more precise.